### PR TITLE
Article for Infix to postfix conversion

### PR DESCRIPTION
**Pull request by SAY-droid427**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes https://github.com/TesseractCoding/NeoAlgo-Docs/issues/196

### CODE REVIEW

**Comment by SAY-droid427** [comment]
@iamrajiv @HarshCasper Please review the PR.

**Review by iamrajiv — APPROVED** [review]

File: docs/data-structures/infix-to-postfix.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+title: Infix to Postfix Conversion
+slug: /data-structures/infix-to-postfix
+---
+
+## Introduction
+
+An infix expression is of the form: `operand1 operator operand2`, whereas a postfix expression is of the form `operand1 operand2 operator`. Converting an infix expression to postfix expression is one of the many applications of stacks due to its 'Last in First Out' policy.
```
Comment (HarshCasper) [inline docs/data-structures/infix-to-postfix.mdx:8]: Why does `operand1 operator operand2` need unnecessary highlighting? Same for `operand1 operand2 operator`

File: docs/data-structures/infix-to-postfix.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
```
Comment (HarshCasper) [inline docs/data-structures/infix-to-postfix.mdx:9]: Add a section to explain the conversion? If possible, prefer using diagrams

File: docs/data-structures/infix-to-postfix.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+## Algorithm
+
+- Initialize a stack.
+- Scan the element from left to right.
+- If the scanned element is a '(' push to stack.
```
Comment (HarshCasper) [inline docs/data-structures/infix-to-postfix.mdx:14]: Use backticks

File: docs/data-structures/infix-to-postfix.mdx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+- If scanned element is a '), pop and output contents of stack till empty or '(' is encountered.
```
Comment (HarshCasper) [inline docs/data-structures/infix-to-postfix.mdx:15]: Use backticks

File: docs/data-structures/infix-to-postfix.mdx [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+### Output
+
+```
+Postfix Expression: ab+cd*+
+```
+
+## Complexity Analysis
+
+- Time Complexity: `O(n^2)`
+
```
Comment (HarshCasper) [inline docs/data-structures/infix-to-postfix.mdx:146]: Remove extra line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: docs/data-structures/infix-to-postfix.mdx (hunk 1 above)
Comment (SAY-droid427) [reply docs/data-structures/infix-to-postfix.mdx]: Changed

File: docs/data-structures/infix-to-postfix.mdx (hunk 2 above)
Comment (SAY-droid427) [reply docs/data-structures/infix-to-postfix.mdx:9]: Used images for explanation

File: docs/data-structures/infix-to-postfix.mdx (hunk 3 above)
Comment (SAY-droid427) [reply docs/data-structures/infix-to-postfix.mdx]: Done

File: docs/data-structures/infix-to-postfix.mdx (hunk 4 above)
Comment (SAY-droid427) [reply docs/data-structures/infix-to-postfix.mdx]: Done

File: docs/data-structures/infix-to-postfix.mdx (hunk 5 above)
Comment (SAY-droid427) [reply docs/data-structures/infix-to-postfix.mdx]: Done

**Comment by SAY-droid427** [comment]
@HarshCasper I have made the changes. Could you please review? And also I can't understand why the checks are failing after resolving the merge conflicts. Thank you!

**Comment by SAY-droid427** [comment]
@HarshCasper Please review this PR, I have made all the changes

**Comment by SAY-droid427** [comment]
@HarshCasper @iamrajiv Please review the changes..

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**